src/sdk/namespace_fs.js

```diff
@@ -721,9 +722,10 @@ class NamespaceFS {
                         if (pos < results.length) {
                             results.splice(pos, 0, r);
                         } else {
-                            const stat = await native_fs_utils.stat_ignore_eacces(this.bucket_path, r, fs_context);
+                            const stat = await native_fs_utils.stat_ignore_eacces_enoent(this.bucket_path, r.key, fs_context);
```

Comment:
I don't feel we want to ignore only these 2 errors... we need to decide on a robust behavior... we will just keep adding error codes all the time

Reply:
Do you suggest that also for the first `stat` we will ignore all the errors?

Reply:
We discussed the error code list, and we divided them into groups (changes in the bucket, permissions/structure, and unexpected errors). In this PR, we will add only the first group.